### PR TITLE
Updated .ruby-version and added tailwind

### PR DESCRIPTION
**Pull request by prio101**
- Added the updated version to the .ruby-version file.
- Added the tailwind
- Added power config.

### CODE REVIEW

File: app/javascript/stylesheets/application.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@import "tailwindcss/base";
+@import "tailwindcss/components";
+@import "tailwindcss/utilities";
```
Comment (dezman) [inline app/javascript/stylesheets/application.scss:3]: newline

File: playwright.config.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+module.exports = {
```
Comment (dezman) [inline playwright.config.js:1]: @prio101 should this be in here? playwright?

File: tests/baked_page.spec.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 const { test, expect } = require('@playwright/test');
 
 test('basic test', async ({ page }) => {
-  await page.goto('http://localhost:3000/baked', {"timeout": 60000});  
+  await page.goto('http://localhost:3000/baked');
```
Comment (dezman) [inline tests/baked_page.spec.js:4]: separate PR

**Review by dezman — APPROVED** [review]

**Review by waseemanwar1 — DISMISSED** [review]

**Comment by dezman** [comment]
@prio101 want to merge this?

**Review by dezman — DISMISSED** [review]

**Review by dezman — APPROVED** [review]

**Pull request merged**